packages/console/style/index.css

```diff
@@ -49,13 +57,18 @@
   max-height: 80%;
   flex: 0 0 auto;
   overflow: auto;
-  border-top: var(--jp-border-width) solid var(--jp-border-color1);
+  border-top: var(--jp-border-width) solid var(--jp-toolbar-border-color);
   padding-top: var(--jp-private-console-cell-padding);
   padding-right: var(--jp-private-console-cell-padding);
   padding-left: var(--jp-private-console-cell-padding);
-  /* The weird 8px bottom padding is needed because of the 2px margin in the panel
-  that lets the shadow in the dock panel show */
-  padding-bottom: 8px;
+  /* We used to have a bottom padding of 8px that was supposedly related to
+   * our implementation of the 2px margin the the panel to let the shadow in the 
+   * dock panel show. But that doesn't seem to be relevant now, so using the 
+   * matching padding. */
+  padding-bottom: var(--jp-private-console-cell-padding);
```

Comment:
these four rules can now be simplified: `padding: var(--jp-private-console-cell-padding)`